doc/migrations/v0.46-v1.0.0.md

```diff
@@ -425,4 +426,4 @@ The observed behavior of dialing peers has been that given a list of supported a
 
 Consequently the previous dial behaviour of dialing all available addresses (up to a concurrency limit) and cancelling any in-flight dials when the first succeeds was a very inefficient use of resources.
 
-Since `libp2p@0.46.10` we have only dialed one address at a time for each peer by setting the default value of the `ConnectionManager`'s `maxParallelDialsPerPeer` option to `1`.  As of `libp2p@1.0.0` this option has been removed.
\ No newline at end of file
+Since `libp2p@0.46.10` we have only dialed one address at a time for each peer by setting the default value of the `ConnectionManager`'s `maxParallelDialsPerPeer` option to `1`.  As of `libp2p@1.0.0` this option has been removed.
```

Comment:
these fixes come from pasting this code into a new file in an `aegir` project (helia) and fixing saving causing auto-fix. can revert if desired.